config/application.rb

```diff
@@ -104,21 +104,17 @@ class Application < Rails::Application
     # end Rails 5 #
 
     # Fix for Psych::DisallowedClass: Tried to load unspecified class
-    # this has to be in "after_initialize" because we need custom classes to be loaded already
-    config.after_initialize do
-      ActiveRecord::Base.yaml_column_permitted_classes = [
-        Symbol,
-        ActiveSupport::HashWithIndifferentAccess,
-        HashWithIndifferentAccess, # rubocop:disable Rails/TopLevelHashWithIndifferentAccess
-        RequestType::Validator::ArrayWithDefault,
-        RequestType::Validator::LibraryTypeValidator,
-        RequestType::Validator::FlowcellTypeValidator,
-        ActionController::Parameters,
-        Set,
-        Range,
-        FieldInfo,
-        Time
-      ]
-    end
+    config.active_record.yaml_column_permitted_classes = Array(config.active_record.yaml_column_permitted_classes) + %w[
```

Comment:
This fix is a combination of:
* Don't set this value in an after_initialize any more (it doesn't work).
* Use strings instead of proper class references to bypass the unloaded classes issue.
* Switch to using `config.active_record` rather than `ActiveRecord::Base` which was an undocumented way of setting the value.